### PR TITLE
User tests

### PR DESCRIPTION
**Pull request by LHJE**
This PR:
- Updates the User tests to correctly test for changing a user's styles

### CODE REVIEW

File: code/api/src/config/params.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       },
       "shoe": {
         "id": 3,
-        "name": "shoe"
+        "name": "Shoe"
```
Comment (Arique1104) [inline code/api/src/config/params.json:14]: Nice catch for standardizing our names.

File: code/api/src/modules/user/tests/userQuery.test.js [hunk 2]
```diff
@@ -46,10 +46,10 @@ describe('test the truth', () => {
   it('updates a users styles', async (done) => {
     const response = await request(server)
       .post('/graphql')
-      .send({query: `mutation {userUpdate { id  } }`})
-      console.log(response.body)
+      .send({query: `mutation { userUpdate( id: 2, primaryStyle: "Racecar Driver", secondaryStyle: "Punk" ) { id primaryStyle secondaryStyle } }`})
```
Comment (Arique1104) [inline code/api/src/modules/user/tests/userQuery.test.js:49]: I really appreciate this strategy for testing this mutation.

File: code/mobile/src/setup/config.json [hunk 3]
```diff
@@ -13,13 +13,25 @@
...
+        "id": 3,
+        "name": "Shoe"
+      },
+      "accessory": {
+        "id": 4,
         "name": "Accessories"
+      },
+      "underpant": {
+        "id": 5,
+        "name": "Underpant"
```
Comment (Arique1104) [inline code/mobile/src/setup/config.json:34]: Nice addition different types of clothes here!

**Review by Arique1104 — APPROVED** [review]
Looks great!  Glad it works!

File: code/api/src/modules/user/tests/userQuery.test.js [hunk 4]
```diff
@@ -46,10 +46,10 @@ describe('test the truth', () => {
...
       .expect(200)
-      expect(response.body.data.users.length).toBe(2);
+      expect(response.body.data.userUpdate.primaryStyle).toBe('Racecar Driver');
+      expect(response.body.data.userUpdate.secondaryStyle).toBe('Punk');
```
Comment (drewwilliams5280) [inline code/api/src/modules/user/tests/userQuery.test.js:52]: Nice catch needing this to be userUpdate

**Review by drewwilliams5280 — APPROVED** [review]

**Pull request merged**